### PR TITLE
Activate IPv6 in CentOS 7 AMI

### PR DESCRIPTION
**Pull request by lloesche**
## High Level Description

IPv6 currently deactivated in the CentOS 7 AMI. This PR makes the AMI run dual stack.

## Related Issues

  - [DCOS-15500](https://jira.mesosphere.com/browse/DCOS-15500) Verify that all DC/OS AMI have dual stack enabled.

### CODE REVIEW

**Comment by lloesche** [comment]
@mesosphere-mergebot label Work In Progress

**Comment by lloesche** [comment]
@mesosphere-mergebot bump-ee

**Comment by lloesche** [comment]
@mesosphere-mergebot label Ready For Review

**Comment by lloesche** [comment]
I tested the new AMI in an IPv6 enabled cluster. I was able to start containers and do all the usual operations. The master and agent nodes had routed v6 addresses and were able to e.g. `ping6 google.com`. For the private agents I had manually added an Egress only gateway to the VPC and adjusted routing tables and security policies accordingly.

File: cloud_images/centos7/create_base_ami.sh [hunk 1]
```diff
@@ -53,7 +53,9 @@ ONBOOT="yes"
 TYPE="Ethernet"
 USERCTL="yes"
 PEERDNS="yes"
-IPV6INIT="no"
+IPV6INIT="yes"
+DHCPV6C="yes"
+NM_CONTROLLED="no"
```
Comment (vespian) [inline cloud_images/centos7/create_base_ami.sh]: Why?

Comment (vespian) [inline cloud_images/centos7/create_base_ami.sh]: I am just curious, do not know the context. NM got really good, basing on my experience and `nmcli` rocks.

Comment (lloesche) [inline cloud_images/centos7/create_base_ami.sh]: It shouldn't have been used in the first place. NetworkManager on EL7 is v1.4.0 (vs. current stable v1.8.0). NM is handy on a laptop moving between multiple networks. Doesn't add any value on a server with static configuration and where you don't need users be able to change the networking. We even turn off firewalld so no gain there either. One less moving part, one less source of errors.

**Comment by mellenburg** [comment]
@lloesche @lingmann @vespian FYI these images for cloud integration tests are now pinned here: https://github.com/mesosphere/dcos-test-utils/blob/master/dcos_test_utils/aws.py

**Pull request merged**